php/commands/site.php

```diff
@@ -994,7 +996,7 @@ public function deploy($args, $assoc_args) {
       'annotation'     => $annotation,
     );
 
-    if ($clone_live_content) {
+    if ($sync_content) {
```

Comment:
Can we make the `--sync-content` flag only valid when performing the operation on the Test environment? I'm not sure if we should just ignore it, or also alert the user that it's an invalid option when deploying Live.